drivers/syslog/note_driver.c

```diff
@@ -41,34 +41,62 @@
 
 #include <sys/types.h>
 #include <sched.h>
+#include <fcntl.h>
 #include <assert.h>
 #include <errno.h>
+#include <string.h>
 
 #include <nuttx/sched_note.h>
+#include <nuttx/sched_note_trace.h>
```

Comment:
Should move the addition of sched_note_trace.h from commit "Add task trace support" to here.
Is the name note_driver.h better than sched_note_trace.h? and put it to nuttx/syslog.

Reply:
I see. I got it.